hack/run-rollouts-manager-e2e-tests.sh

```diff
@@ -178,7 +178,7 @@ sanity_test_metrics_data() {
 
   fi
 
-  if [[ "`expr $DELTA_ERROR_RECONCILES \> 60`" == "1" ]]; then
+  if [[ "`expr $DELTA_ERROR_RECONCILES \> 70`" == "1" ]]; then
```

Comment:
Just as a small hint, if you want to test if that variable is greater than 70, you can also use the much simpler following construct :)

```
if [[ "$DELTA_ERROR_RECONCILES" -gt 70 ]]; then
```